NeuroID/src/main/java/com/neuroid/tracker/service/NIDValidationService.kt

```diff
@@ -45,4 +45,29 @@ internal class NIDValidationService(
 
         return true
     }
+
+    fun scrubIdentifier(identifier: String): String {
+        val emailRegex = Regex("[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}")
+        val ssnRegex = Regex("\\b\\d{3}-\\d{2}-\\d{4}\\b")
+
```

Comment:
should the regex include any 9 digit possible values without the hyphens?

Reply:
That's a fair question, tbh this was just a quick check for SSN, even if they do set it we would have no way of really knowing that it's an SSN vs. a 9 digit identifier that the customer uses which is why I didn't add more vigorous regex

Reply:
sounds good! Looks Good to me!